hr_employee_private_wizard/wizard/hr_employee_private_wizard.py

```diff
@@ -0,0 +1,103 @@
+# © 2020 - Today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from lxml import etree
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using etree to parse untrusted XML data is known to be vulnerable to XML attacks. Replace etree with the equivalent defusedxml package.](https://app.codacy.com/manual/numigi/odoo-hr-addons/pullRequest?prid=5358386)